packages/react-native/React/DevSupport/RCTInspectorUtils.mm

```diff
@@ -25,7 +25,8 @@ + (CommonHostMetadata *)getHostMetadata
   // macOS does not support UIDevice. Use System Configuration. This API
   // returns a nullable value, but is non-blocking (compared with
   // `[NSHost currentHost]`) and is ideal since deviceName is optional.
-  NSString *deviceName = (__bridge NSString *)SCDynamicStoreCopyComputerName(nil, nil);
+//  NSString *deviceName = (__bridge NSString *)SCDynamicStoreCopyComputerName(nil, nil);
+    NSString *deviceName = @"";
```

Comment:
Fix: This needs you to link against the System Configuration framework, which upstream doesn't currently do.

Reply:
Will follow up post.